src/modules/spaces/services/spaces.service.ts

```diff
@@ -24,7 +24,9 @@ export class SpacesService {
 			id: get(graphQlSpace, 'id'),
 			maintainerId: get(graphQlSpace, 'schema_maintainer.schema_identifier'),
 			name: get(graphQlSpace, 'schema_maintainer.schema_name'),
-			description: get(graphQlSpace, 'schema_description'),
+			description: !isEmpty(graphQlSpace.schema_description)
+				? graphQlSpace.schema_description
+				: get(graphQlSpace, 'schema_maintainer.information[0].description'),
```

Comment:
Avoid "not" in if else conditions:

```suggestion
			description: isEmpty(graphQlSpace.schema_description)
				? get(graphQlSpace, 'schema_maintainer.information[0].description'),
				: graphQlSpace.schema_description
```

It's easier to interpret: 
```
if description is empty
do
else do
```

than it is to interpret:
```
if description is not empty
do
else do
```